Assets/Scenes/editor.unity

```diff
@@ -1202,6 +1206,29 @@ PrefabInstance:
       objectReference: {fileID: 0}
     m_RemovedComponents: []
   m_SourcePrefab: {fileID: 100100000, guid: 44790cf12ccd011459f749712f90a90b, type: 3}
+--- !u!1 &5677096275816853217 stripped
+GameObject:
+  m_CorrespondingSourceObject: {fileID: 5677096275094490002, guid: 44790cf12ccd011459f749712f90a90b, type: 3}
+  m_PrefabInstance: {fileID: 5677096275816853216}
+  m_PrefabAsset: {fileID: 0}
+--- !u!114 &5677096275816853218
+MonoBehaviour:
```

Comment:
Please avoid using overrides. Try to apply as much as possible to the respective prefabs

Reply:
I just updated the prefab, adding the new script, hope it solves this problem.